.github/workflows/ci.yml

```
@@ -0,0 +1,74 @@
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      fullSemVer: ${{ steps.gitversion.outputs.fullSemVer }}
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)